docker-compose-dev.yml

```diff
@@ -12,8 +12,25 @@ services:
       - THRDS_CONNGRACEPERIOD
       - THRDS_KEEPALIVEINTERVAL
       - THRDS_ENABLENETPUBSUB
-      - THRDS_DEBUG
+      - THRDS_MONGOURI=mongodb://mongo:27017
+      - THRDS_MONGODATABASE
+      - THRDS_DEBUG=true
     ports:
       - "4006:4006"
       - "127.0.0.1:6006:6006"
       - "127.0.0.1:6007:6007"
+    depends_on:
+      - mongo
+  mongo:
+    image: mongo:latest
+    expose:
+      - "27017"
+    ports:
+      - "127.0.0.1:27017:27017"
+    command:
+      - /bin/bash
+      - -c
+      - |
+        /usr/bin/mongod --fork --logpath /var/log/mongod.log --bind_ip_all --replSet rs0
+        mongo --eval 'rs.initiate({_id: "rs0", version: 1, members: [{ _id: 0, host: "mongo:27017" }]})'
+        tail -f /var/log/mongod.log
```

Comment:
Cool, what we chated in the other PR. 👍🏼

Reply:
Yep, I added a little test util package in `go-ds-mongo`... makes it easy to start the replica set for tests: https://github.com/textileio/go-ds-mongo/blob/master/mongo_test.go#L20